dev-requirements.txt

```diff
@@ -0,0 +1,10 @@
+pylint~=2.3
```

Comment:
Should this file have the word `constraints` in it as well? `dev-constraints.txt` maybe? Due to use using it as constraints in `tox`.

Reply:
Well, if you use the `install --with-dev-deps` or `develop` commands, it will be used as a requirements file.